### ISSUE TITLE
Border or shadow around the image

### ISSUE THREAD

**Issue by zu1in** [jurplel/qView#447]
It would be cool to have a 1px black (or even adjustable color) **border** or **shadow** to be able to see bounds of **transparent** images. Really miss this feature. Other than that -- a perfect viewer! :)

**Comment by jurplel** [comment]
Good idea! I think i actually planned this at some point as well. I will consider in the future but I have more pressing things to fix first.

**Comment by Spuner** [comment]
It would be nice if they did it like in Windows Photo Viewer - so that it would be impossible to go beyond the image borders.

**Comment by jurplel** [comment]
> It would be nice if they did it like in Windows Photo Viewer - so that it would be impossible to go beyond the image borders.

I think there are a lot of apps that do this—it's definitely more common than qView's style. I imagine an option will be added later when graphicsView is rewritten.
